src/cloud/reference/cli-ref-topic.md

```diff
@@ -117,7 +157,7 @@ Help:
 
 ## Update Magento Cloud CLI
 
-The Magento Cloud CLI checks for available updates upon login, but you can also check for updates using the `self:update` command. If there is an update available, follow the instructions and enter `Y` for _yes_ to update the CLI.
+The Magento Cloud CLI checks for available updates when you log in, but you can check for updates using the `self:update` command. If there is an update available, follow the instructions and enter `Y` for _yes_ to update the CLI.
```

Comment:
The "enter `Y` for _yes_ is awkward, and I think unnecessary since the prompt provides the instructions.

```suggestion
The Magento Cloud CLI checks for available updates when you log in, but you can check for updates using the `self:update` command. If there is an update available, follow the instructions to update the CLI.
```